### PR TITLE
fix: ref resolving in dev portal

### PR DESCRIPTION
**Pull request by mpodlasin**
Relates to Qualtrics issues @philsturgeon was mentioning on slack and during standup.

Simple mistake on our part.

I am not providing tests here, because I believe this should be a part of e2e suite - I left a comment in https://github.com/stoplightio/elements/issues/1141.

### CODE REVIEW

**Review by philsturgeon — APPROVED** [review]

**Comment by philsturgeon** [comment]
Let's tag a beta2 so we can see if this helps Qualtrics and Calendly who also reported this issue.

File: packages/elements-dev-portal/src/components/NodeContent/index.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ export const NodeContent = ({ node, Link, hideTryIt, hideMocking }: NodeContentP
                 nodeData={node.data}
                 nodeTitle={node.title}
                 hideTryIt={hideTryIt}
+                useNodeForRefResolving
```
Comment (marcelltoth) [inline packages/elements-dev-portal/src/components/NodeContent/index.tsx:42]: Shouldn't this be true by default? 🤔

**Review by marcelltoth — APPROVED** [review]

File: packages/elements-dev-portal/src/components/NodeContent/index.tsx (hunk 1 above)
Comment (mpodlasin) [reply packages/elements-dev-portal/src/components/NodeContent/index.tsx:42]: I am ok either way. 🤷

Comment (lottamus) [reply packages/elements-dev-portal/src/components/NodeContent/index.tsx:42]: I'm for making it true by default. Is there a use case for not resolving refs?

Comment (marcelltoth) [reply packages/elements-dev-portal/src/components/NodeContent/index.tsx:42]: There must be as @mpodlasin  added it as a prop and not by default. @mpodlasin do you know the reason maybe?

**Pull request merged**